qemu/tests/avic_test.py

```diff
@@ -0,0 +1,47 @@
+from avocado.utils import process, cpu
+from virttest import env_process
+from provider.cpu_utils import check_cpu_flags
+
+
+def run(test, params, env):
+    """
+    avic test:
+    1) Turn on avic on Genoa host
+    2) Check if reading kvm_amd parameter crash host
+    3) Launch a guest
+    3) Check no error in guest
+    4) Restore env, turn off avic
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    if cpu.get_cpu_vendor_name() != 'amd':
+        test.cancel("This test is supposed to run on AMD host")
+    flags = params["flags"]
+    check_host_flags = params.get_boolean("check_host_flags")
+    if check_host_flags:
+        check_cpu_flags(params, flags, test)
+
+    unload_cmd = params["unload_cmd"]
+    load_cmd = params["load_cmd"]
```

Comment:
`kvm_probe_module_parameters = "avic=1"`

Reply:
> `kvm_probe_module_parameters = "avic=1"`

What's this for? I add the value in Line 34.

Reply:
> > `kvm_probe_module_parameters = "avic=1"`
> 
> What's this for? I add the value in Line 34.

Answer is upon comment, resolve this.